### PR TITLE
Updating the api.proto

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ $ pip install -r requirements.txt
 Run tests:
 
 ```
-$ make test
+$
```
Comment (DimaCantemir) [inline README.md]: was this on purpose ?

Comment (aspin) [inline README.md]: revert this :laugh:

**Review by aspin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tinystarinagalaxy) [reply README.md]: ok

Comment (tinystarinagalaxy) [reply README.md]: done

**Pull request merged**